src/class-parsely.php

```diff
@@ -640,6 +640,11 @@ private function are_credentials_managed(): bool {
 			return false;
 		}
 
+		if ( isset( $credentials['is_managed'] ) &&
+				is_bool( $credentials['is_managed'] ) ) {
+			return $credentials['is_managed'];
+		}
+
```

Comment:
We can remove all of the code below and just return `false`.

If `site_id` and `api_secret` is set then `is_managed` will also always be set so the code below is unreachable.

Reply:
So to conclude, our only criterion is whether `is_managed` is set. We don't care about anything else. Correct?

Reply:
Correct

Reply:
Updated. Let me know if it suits the needs.

Reply:
Tests are failing after the update, looking into it.